### DISCUSSION TITLE
Running over parallels VM

### DISCUSSION THREAD

**Discussion by ronlivne** [ichi4096/vivado-on-silicon-mac#26] [Q&A] [answered]
Is it possible to run this solution over parallels vm with Rosetta 2 (parallels 19 and above).

**Comment by sapertuz** [comment]
do you have any updates on this?

**Comment by ichi4096** [comment, marked as answer]
No. Docker has an environment where all binaries (except the kernel and kernel modules) are x64 and `uname` also returns the x64 architecture. Other solutions, like UTM and Parallels, do not do that, which Vivado will complain about.
